FirebaseRemoteConfig/Tests/HermeticAPI/APITests.swift

```diff
@@ -130,6 +125,41 @@ class APITests: XCTestCase {
     waitForExpectations()
   }
 
+  // Test New API.
+  func testUnchangedActivateWillFlag() {
+    let expectation = self.expectation(description: #function)
+    config.fetch { status, error in
```

Comment:
Will this also help us adding ResultType?

Reply:
Yep, but adding a Swift extension for RemoteConfig is out of scope for now.